profiles/debug_way.lua

```diff
@@ -0,0 +1,44 @@
+--
+-- Fetch a way from the OpenStreetMap API and run the given profile over it.
+--
+-- You'll need to install luasec and xml2lua first:
+--    > luarocks-5.1 install xml2lua
+--    > luarocks-5.1 install luasec
+-- [may require admin privileges]
+-- 
+-- Then to test way 2606296 using the foot profile:
+--    > lua5.1 debug_way.lua foot 2606296
```

Comment:
My system (Ubuntu) currently does not have `lua5.1` installed but `lua5.2` which is also accessible via the `lua` generic command. If the same apply across other systems, pointing to simply `lua` would make it more generic.

Reply:
I guess it applies not only for ubuntu, so indeed, this needs to be just `lua`

Reply:
No problem, changed (blame Homebrew for this one ;) ).